packages/sdk/src/protocols/light_push.ts

```diff
@@ -90,18 +90,20 @@ class LightPushSDK extends BaseProtocolSDK implements ILightPushSDK {
         }
         if (failure) {
           if (failure.peerId) {
-            try {
-              await this.renewPeer(failure.peerId);
-              log.info("Renewed peer", failure.peerId.toString());
-            } catch (error) {
-              log.error("Failed to renew peer", error);
-            }
+            // schedule renewal as we don't need to wait for it
+            this.renewPeer(failure.peerId)
```

Comment:
@danisharora099 wdyt about that change?

Reply:
rolling back that change as it decreases testability, will followup (cc @danisharora099 )

Reply:
added as follow up here https://github.com/waku-org/js-waku/issues/2078#issuecomment-2318741996